<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompilerSetDirectoriesWidget</class>
 <widget class="QWidget" name="CompilerSetDirectoriesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnAdd">
        <property name="toolTip">
         <string>Add</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/newlook24/002-add.png</normalon>
         </iconset>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDelete">
        <property name="toolTip">
         <string>Remove</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/newlook24/008-close.png</normalon>
         </iconset>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRemoveInvalid">
        <property name="toolTip">
         <string>Remove Invalid</string>
        </property>
        <property name="text">
         <string>Remove Invalid</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/newlook24/007-bughlp.png</normalon>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
